lang/common/bridge/avro/Copyright.txt

```diff
@@ -0,0 +1,16 @@
+// Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
Microsoft Avro does not add Apache copyrights to the generated C# code like Apache Avro does to the generated Java code which causes build errors. I concatenate this file into the Microsoft Avro generated files from the powershell script. I can move this into the powershell script. I took the shortest path to a pull request due to the need to get the bridge split ASAP.